### PR TITLE
Feature/add coach profile table/rename

### PR DESCRIPTION
**Pull request by ExxZzyy**
Feature/add coach profile table

### CODE REVIEW

**Comment by ExxZzyy** [comment]
image

**Comment by ExxZzyy** [comment]
image
Updated theme

File: exadel-trainee-app/src/app/models/coach-profile/coach-profile.component.ts [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+  dataSource: MatTableDataSource<UserData>;
+
+  @ViewChild(MatPaginator) paginator!: MatPaginator;
+
+  @ViewChild(MatSort) sort!: MatSort;
+
+  constructor() {
+    // eslint-disable-next-line @typescript-eslint/no-use-before-define
+    const users = Array.from({ length: 100 }, (_, k) => createNewUser(k + 1));
```
Comment (RomPavel) [inline exadel-trainee-app/src/app/models/coach-profile/coach-profile.component.ts:60]: Just put it on the same level with displayedColumns definition and lets remove eslint-disable

Comment (ExxZzyy) [reply exadel-trainee-app/src/app/models/coach-profile/coach-profile.component.ts:29]: https://github.com/Inunor/internship/pull/64/commits/b2b255d6e170ab9473631c60dbe682cd8dd2be5b
done

File: exadel-trainee-app/src/app/models/coach-profile/coach-profile.component.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  applyFilter(event: Event) {
+    const filterValue = (event.target as HTMLInputElement).value;
+    this.dataSource.filter = filterValue.trim().toLowerCase();
+
+    if (this.dataSource.paginator) {
+      this.dataSource.paginator.firstPage();
+    }
+  }
+
+  OnClick(id: number) {
```
Comment (Inunor) [inline exadel-trainee-app/src/app/models/coach-profile/coach-profile.component.ts:46]: onClick :) angular uses camelCase notation

File: exadel-trainee-app/src/app/models/coach-profile/coach-profile.component.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
+import { Component, AfterViewInit, ViewChild } from '@angular/core';
+import { MatSort } from '@angular/material/sort';
+import { MatPaginator } from '@angular/material/paginator';
+import { MatTableDataSource } from '@angular/material/table';
+import { createNewUser } from 'src/mocks/users-utils.mock';
+
+export interface UserData {
```
Comment (Inunor) [inline exadel-trainee-app/src/app/models/coach-profile/coach-profile.component.ts:7]: https://github.com/Inunor/internship/pull/70/files#diff-1f0244162fd150398b6cafbd2fe838d31f6de4df9ec842e0d50d7d49bba4634f
Please cooperate somehow, you already have some kind of User inteface

**Review by Inunor — APPROVED** [review]

**Pull request merged**